openforcefield/utils/serialization.py

```diff
@@ -354,6 +364,7 @@ def from_messagepack(cls, serialized):
         d = msgpack.loads(serialized, raw=False)
         return cls.from_dict(d)
 
+    @requires_package("xmltodict")
```

Comment:
`xmltodict` is needed to OFFXML I/O, so it's one of our least optional dependencies. 
```suggestion
```